lib/filter/enemy.js

```diff
@@ -54,9 +54,10 @@ exports.update = function(json) {
         rows.push(currentEnemy.params[0].disp_name);
         rows.push(currentEnemy.lv);
         rows.push(currentEnemy.uid);
-        rows.push(currentEnemy.init_hp + '/' + currentEnemy.max_hp);
 
         for (var param in json[round].enemy[enemy].children[part].params) {
+          currentEnemy.init_hp = config.get('enemy.stats.hp') || currentEnemy.init_hp;
```

Comment:
I think there is not need, to put this in this loop. We have to set it only once.